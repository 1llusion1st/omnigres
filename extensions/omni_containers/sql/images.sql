SET omni_containers.fixtures TO 'fixtures';
\x ON;
SELECT * FROM omni_containers.docker_images;