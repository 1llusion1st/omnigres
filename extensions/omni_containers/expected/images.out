-- Ensure we pull an image
SET client_min_messages TO WARNING;
WITH container AS (SELECT * FROM omni_containers.docker_container_create('ghcr.io/yrashk/psql', cmd => $$true$$,
                                                      wait => true, pull => true) AS id)
SELECT true AS result FROM container;
 result 
--------
 t
(1 row)

SET client_min_messages TO NOTICE;
\x ON
SELECT true AS result FROM omni_containers.docker_images WHERE 'ghcr.io/yrashk/psql:latest' = any(repo_tags);
-[ RECORD 1 ]
result | t

